static-ips/static-ips.tf

```diff
@@ -16,30 +16,12 @@ resource "google_compute_address" "ui" {
   region   = each.value["clusterRegion"]
 }
 
-resource "google_compute_address" "api" {
```

Comment:
We added the special reserved UI & API CloudNAT IP addresses to some firewall rules.  Are we certain that we want to lose the API address here (thinking for future needs perhaps)?

What do you think @billiford?

Reply:
API address should stay otherwise GHE cannot talk to Spinnaker for webhook commits

Reply:
switched ui dns stuff to api a840ca9